utils/spdx/src/main/resources/simple-license-mapping.yml

```diff
@@ -23,41 +23,22 @@
 ---
 
 AFLv2.1: AFL-2.1
-ALv2: Apache-2.0
```

Comment:
This further strictens the check to not tolerate differences in trailing zeros when extract digits. This avoids the theoretical issue that e.g. Apache2 could either refer to `Apache-2.0` or `Apache-2.1` *if* the latter came out at some point.